src/topology/algebra/module/finite_dimension.lean

```diff
@@ -347,6 +347,9 @@ begin
   { simp only [map_sub, map_add, ← comp_apply f g, hg, id_apply, sub_add_cancel] }
 end
 
+instance can_lift_continuous_linear_map : can_lift (E →ₗ[𝕜] F) (E →L[𝕜] F) coe ⊤ :=
```

Comment:
Does the `⊤` here generate a useless side-goal?

Reply:
No, you can close it using `lift f to E →L[𝕜] F using trivial`.

Reply:
If you don't close it on the fly, is the goal `true` or `⊤ f`?

Reply:
`⊤ f`

Reply:
If you use a lambda, do you get `true` by itself?

Reply:
Yeah indeed that's better.